pkg/controller/test-images-distributor/test_images_distributor.go

```diff
@@ -226,11 +226,16 @@ func (r *reconciler) reconcile(req reconcile.Request, log *logrus.Entry) error {
 	sourceImageStreamTag := &imagev1.ImageStreamTag{}
 	if err := r.registryClient.Get(r.ctx, decoded, sourceImageStreamTag); err != nil {
 		if apierrors.IsNotFound(err) {
-			log.Debug("Source imageStreamTag not found")
+			log.Trace("Source imageStreamTag not found")
```

Comment:
Do we know what causes this to happen?  It is worth adding a comment.

Reply:
This is needed because whats put in the workqueue is always a string identifier to allow for deduplication. However this means that by the time the object gets reconciled, it might not exist anymore. This check is needed in all controllers so not sure if it warrants a comment.